src/hpr_multi_buy.erl

```diff
@@ -70,6 +71,13 @@ cleanup(Duration) ->
     end),
     ok.
 
+-spec make_key(hpr_packet_up:packet(), hpr_route:route()) -> binary().
+make_key(PacketUp, Route) ->
+    crypto:hash(sha256, <<
+        (hpr_packet_up:phash(PacketUp))/binary,
```

Comment:
For the record, if this uses erlang:phash2 under the hood, you should know there is 1 in 27,000 chance of collision returning the same value. Not sure if you care about that - if this is a long lived look up value, I would find another way to get a more durable hash.